Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -1126,9 +1126,10 @@ void TerminalWidget::drop_event(GUI::DropEvent& event)
             if (!first)
                 send_non_user_input(" "sv.bytes());
 
-            if (url.protocol() == "file")
-                send_non_user_input(url.path().bytes());
-            else
+            if (url.protocol() == "file") {
+                auto text = String::formatted(url.path().contains(' ') ? "\"{}\"" : "{}", url.path());
```

Comment:
What if it includes quotes? or `$`? or `*?`? Or...

Reply:
Yes I think that is a valid and obvious extension of this. I didn't immediately find an existing escape function and and this covered my current use case of another thing I was working with.

If you want I can pick this PR up another day to complete.

Reply:
imo the Shell should be doing this, given its knowledge of how it interprets strings.
I'd suggest adding a `on_paste` (or equivalent) to LibLine, and setting that in the Shell to insert a properly escaped version of the pasted thing if it looks like a file path.